eg-next/examples/primitives-fill.rs

```diff
@@ -33,21 +33,21 @@ where
         .into_styled(style)
         .draw(target)?;
```

Comment:
I think the `stroke_off_fill_of` rectangle clips the lower left corner of the `stroke` triangle.